Description: correct includes
 This part of the code was moved into it's own project and was packaged
 separately by me. To make the build process work, this small fix is
 necessary.
Author: Philipp Huebner <debalance@debian.org>

Index: ejabberd/src/mod_sip_proxy.erl
===================================================================
--- ejabberd.orig/src/mod_sip_proxy.erl
+++ ejabberd/src/mod_sip_proxy.erl
@@ -40,7 +40,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 -define(SIGN_LIFETIME, 300). %% in seconds.
 
Index: ejabberd/src/mod_sip.erl
===================================================================
--- ejabberd.orig/src/mod_sip.erl
+++ ejabberd/src/mod_sip.erl
@@ -38,7 +38,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 %%%===================================================================
 %%% API
Index: ejabberd/src/mod_sip_registrar.erl
===================================================================
--- ejabberd.orig/src/mod_sip_registrar.erl
+++ ejabberd/src/mod_sip_registrar.erl
@@ -37,7 +37,7 @@
 
 -include("ejabberd.hrl").
 -include("logger.hrl").
--include_lib("esip/include/esip.hrl").
+-include_lib("p1_sip/include/esip.hrl").
 
 -define(CALL_TIMEOUT, timer:seconds(30)).
 -define(DEFAULT_EXPIRES, 3600).
Index: ejabberd/include/jlib.hrl
===================================================================
--- ejabberd.orig/include/jlib.hrl
+++ ejabberd/include/jlib.hrl
@@ -22,7 +22,7 @@
 -ifdef(NO_EXT_LIB).
 -include("fxml.hrl").
 -else.
--include_lib("fast_xml/include/fxml.hrl").
+-include_lib("p1_xml/include/fxml.hrl").
 -endif.
 
 -define(STANZA_ERROR(Code, Type, Condition),
